internal/services/device_data.go

```diff
@@ -73,49 +80,7 @@ var activityFieldExists = []esquery.Mappable{
 	esquery.Exists("data.throttlePosition"),
 }
 
-func (c *elasticDeviceDataClient) GetLastActivity(userDeviceID string) (lastActivity time.Time, seen bool, err error) {
```

Comment:
Oh did I kill this from the controller? I think I did. Good to get rid of, thanks.